package.json

```diff
@@ -7,25 +7,33 @@
   "main": "dist/index.js",
   "scripts": {
     "build": "rimraf dist && tsc",
-    "cache:clear": "./set-project-vars.sh --no-cache",
-    "deploy": "npm run deploy:via:tf",
-    "deploy:via:gcloud": "./deploy-via-gcloud.sh",
-    "deploy:via:terraform": "npm run deploy:via:tf",
-    "deploy:via:tf": "terraform -chdir=infra apply",
+    "cache:clear": "./bin/get-project-vars.sh --invalidate-cache",
+    "deploy:function:prod": "./bin/deploy-via-gcloud.sh prod",
+    "deploy:function:staging": "./bin/deploy-via-gcloud.sh staging",
+    "deploy:prod": "terraform -chdir=infra workspace select prod && terraform -chdir=infra apply",
+    "deploy:staging": "terraform -chdir=infra workspace select staging && terraform -chdir=infra apply",
+    "destroy:prod": "terraform -chdir=infra workspace select prod && terraform -chdir=infra destroy",
+    "destroy:staging": "./bin/destroy-project.sh staging",
     "dev": "nodemon --watch 'src/**/*.ts' --exec 'npm run build && npm run start'",
     "gcp-build": "npm run build",
     "generate:env": "terraform -chdir=infra apply -target=local_file.env_file",
-    "logs": "npm run logs:function",
-    "logs:function": "./get-function-logs.sh",
-    "logs:function:url": "./get-function-logs-url.sh",
-    "logs:jobs": "./get-job-logs.sh",
-    "logs:jobs:url": "./get-job-logs-url.sh",
+    "logs": "npm run logs:function:staging",
```

Comment:
should we remove this?

Reply:
personally, it's my most frequently used command actually :D

like it as a shorthand because in most cases the staging logs is probably what we want when developing this